app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by id: session[:user_id]
```

Comment:
`||=` là gì em

Reply:
- Assignment operator with OR logical
- x ||= y (y will assign to x if x is nil or false)

Reply:
Coppy à em. Toàn chơi tiếng anh thế người việt với nhau bắt anh đọc tiếng anh, t đỗi t ko check pull nữa nhé

Reply:
Em tự viết á anh, không có copy
Tại em viết tiếng việt sẽ dài dòng mà lại gây khó hiểu hoặc bị hiểu nhầm sang ý khác
Lần sau trả lời comment sẽ viết tiếng việt, mong anh thông cảm ạ